### ISSUE TITLE
Adguard Home sometimes not starting on router restart

### ISSUE THREAD

**Issue by privacyguy123** [jumpsmm7/Asuswrt-Merlin-AdGuardHome-Installer#195]
Despite logs saying it is active.

Only started happening after installing and removing avahi-utils and it's associated dependencies. :S

**Comment by jumpsmm7** [comment]
Yea you stumped me with this one. Let me know if you discover anything else. Sounds like there is something causing conflict with adguardhome starting. Too bad I haven't been able to reproduce tge issue. Please feel free to share what you find here.

**Comment by privacyguy123** [comment]
Yes unfortunately I suspect a conflict between other scripts/addons. Is there a way to run AG on this router without touching/"taking over" the original dnsmasq file?

**Comment by jumpsmm7** [comment]
> Yes unfortunately I suspect a conflict between other scripts/addons. Is there a way to run AG on this router without touching/"taking over" the original dnsmasq file?

Not when using this installer; however, you can install entwares version of adguardhome and craft your own way of running adguardhome without it having to use Port 53. What this means is that instead of DNSMASQ switching to a different port, it will instead use port 53 and forward request to AdGuardHome which resides on another port. This can create problems because only one client will be observable in the client query log of AdGuardHome (the router). you would no see clients individualized traffic.
